### PR TITLE
Jen-Mei Wu

### PR DESCRIPTION
**Pull request by jenmei**
Added signature.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -245,7 +244,7 @@ <h1 class="project-name">neveragain.tech</h1>
   <li><a href="https://responsibledata.io/">Responsible Data Forum</a> (data ethics)
   <li><a href="https://datasociety.net/">Data & Society</a> (research on social impact of data)
   <li><a href="http://www.acm.org/about-acm/acm-code-of-ethics-and-professional-conduct">ACM Software Engineering Code of Ethics and Professional Practice</a> (and <a href="http://ethics.acm.org/code-of-ethics/code-2018/">revisions committee</a>)
-  <li><a href="https://ssd.eff.org/">EFF&rsquo;s Surveillance Self-Defense Guide</a> 
+  <li><a href="https://ssd.eff.org/">EFF&rsquo;s Surveillance Self-Defense Guide</a>
```
Comment (hurtstotouchfire) [inline index.html]: Hey there, can you submit a separate PR for this? We would have a larger discussion about adding a resource, but you're welcome to sign.

File: index.html [hunk 2]
```diff
@@ -172,8 +172,7 @@ <h1 class="project-name">neveragain.tech</h1>
 <li><a href="https://diracdeltas.github.io">Yan Zhu, Security Engineer</a>
 
 <!-- Append an <li> and your own name and position/affiliation below. -->
-
-  
+<li>Jen-Mei Wu, <a href="http://oaklandmakerspace.com/">Liberating Ourselves Locally</a>
```
Comment (hurtstotouchfire) [inline index.html]: Hey there, we realized we need to do these in alphabetical order to minimize conflicts. Can you update this? Let me know if you need any help figuring out how to rebase / amend.

See https://github.com/neveragaindottech/neveragaindottech.github.io/pull/23 for updated README instructions.

File: index.html (hunk 1 above)
Comment (zestyping) [inline index.html]: This looks like just a spacing change so I'm going to close this PR.

File: index.html (hunk 2 above)
Comment (zestyping) [inline index.html]: Jen-Mei, I've added your name for you.  Thanks for signing!
